open_schools_platform/organization_management/circles/services.py

```diff
@@ -105,3 +113,71 @@ def is_organization_related_to_student_profile(organization_id: str, student_pro
 def convert_str_to_point(string: str):
     res = re.findall(r"\d+\.\d+", string)
     return Point(float(res[0]), float(res[1]), srid=4326)
+
+
+def setup_scheduled_notifications(circle: Circle, notification_delays: list[timedelta]):
+    tasks = PeriodicTask.objects.filter(args=f'["{circle.id}"]', task=send_circle_lesson_notification.name)
+    CrontabSchedule.objects.filter(id__in=list(map(lambda task: task.crontab.id, tasks))).delete()
```

Comment:
should we delete it hard?

Reply:
I think so, because the cron schedule doesn't contain unique data, everything that is important is in circle model.